main.c

```diff
@@ -819,15 +847,30 @@ main(int argc, char **argv)
 						return EXIT_FAILURE;
 					}
 
-					if (percentage > maximum) {
+					if (percentage > maximum && !allow_overflow) {
 						wob_log_error("Received value %ld is above defined maximum %ld", percentage, maximum);
 						if (!hidden) wob_hide(&app);
 						wob_destroy(&app);
 
 						return EXIT_FAILURE;
 					}
+					else if (percentage > maximum && allow_overflow) {
+						effective_color = colors.overflow;
+						if (no_wrap) {
+							percentage = 100;
+						}
+						else {
+							percentage %= maximum;
+						}
+					}
 
-					wob_log_info("Received input { value = %ld, bg = %#x, border = %#x, bar = %#x }", percentage, colors.background, colors.border, colors.bar);
+					wob_log_info(
+						"Received input { value = %ld, bg = %#x, border = %#x, bar = %#x, overflow = %#x }",
+						percentage,
+						colors.background,
+						colors.border,
+						colors.bar,
+						colors.overflow);
```

Comment:
This shouldn't be here. Overflow color is not received via stdin. Maybe something like:

```
Received input { value = %ld, bg = %#x, border = %#x, bar = %#x, overflow = true|false }
```

Reply:
I changed it to show whether or not it's currently overflowed, and then show the color set it's currently using.